src/js/components/search/filters/agency/SelectedAgencies.jsx

```diff
@@ -22,6 +22,13 @@ export default class SelectedAgencies extends React.Component {
             const agency = entry[1];
             let label = agency.subtier_agency.name;
 
+            if (agency.agencyType === 'subtier') {
```

Comment:
Can you handle if the abbreviation is an empty string? Right now it displays as `Agency ()`.